src/Persistence/Initialization/Updates/FixChaosMixesPlugInSeason6.cs

```diff
@@ -0,0 +1,422 @@
+// </copyright>
+
+namespace MUnique.OpenMU.Persistence.Initialization.Updates;
+
+using System.Runtime.InteropServices;
+using MUnique.OpenMU.DataModel.Attributes;
+using MUnique.OpenMU.DataModel.Configuration;
+using MUnique.OpenMU.DataModel.Configuration.ItemCrafting;
+using MUnique.OpenMU.DataModel.Configuration.Items;
+using MUnique.OpenMU.GameLogic.Attributes;
+using MUnique.OpenMU.GameLogic.PlayerActions.Craftings;
+using MUnique.OpenMU.Network;
+using MUnique.OpenMU.Persistence.Initialization.Items;
+using MUnique.OpenMU.PlugIns;
+
+/// <summary>
+/// This update fixes the Chaos Weapon, First Wings, Dinorant, Item Level Upgrade, Second Wings, Third Wings, Cape, SD Potions, Guardian Option, and Secromicon crafting settings; Blue Fenrir (Protect) damage decrease option value; Wizard's Ring wizardry option; lvl 380 item guardian options for Summoner and Rage Fighter.
+/// </summary>
+[PlugIn(PlugInName, PlugInDescription)]
+[Guid("EFD7EA69-56AE-48A3-ACE2-1C3B5B87780A")]
+public class FixChaosMixesPlugInSeason6 : FixChaosMixesPlugInBase
+{
+    /// <summary>
+    /// The plug in name.
+    /// </summary>
+    private new const string PlugInName = "Fix Chaos Mixes Settings And Several Options";
+
+    /// <summary>
+    /// The plug in description.
+    /// </summary>
+    private new const string PlugInDescription = "This update fixes the Chaos Weapon, First Wings, Dinorant, Item Level Upgrade, Second Wings, Third Wings, Cape, SD Potions, Guardian Option, and Secromicon crafting settings; Blue Fenrir (Protect) damage decrease option value; Wizard's Ring wizardry option; lvl 380 item guardian options for Summoner and Rage Fighter.";
+
+    /// <inheritdoc />
+    public override string Name => PlugInName;
+
+    /// <inheritdoc />
+    public override string Description => PlugInDescription;
+
+    /// <inheritdoc />
+    public override string DataInitializationKey => VersionSeasonSix.DataInitialization.Id;
+
+    /// <inheritdoc />
+    public override UpdateVersion Version => UpdateVersion.FixChaosMixesSeason6;
+
+    /// <inheritdoc />
+    protected override async ValueTask ApplyAsync(IContext context, GameConfiguration gameConfiguration)
+    {
+        // Dark horse spirit and raven spirit drop item groups id fix (do this first because it persists changes to DB)
+        if (gameConfiguration.Items.Single(id => id.Name == "Spirit") is { } spirit)
+        {
+            spirit.MaximumItemLevel = 1;
+            var maps = gameConfiguration.Maps;
+            if (gameConfiguration.DropItemGroups.Single(dig => dig.Description == "Dark Horse Spirit") is { } horseGroup
+                && gameConfiguration.DropItemGroups.Single(dig => dig.Description == "Dark Raven Spirit") is { } ravenGroup)
+            {
+                ClearDropItemGroup(horseGroup);
+                ClearDropItemGroup(ravenGroup);
+                await context.SaveChangesAsync().ConfigureAwait(false);
```

Comment:
Please remove `context.SaveChangesAsync`

Reply:
I'm afraid it is not possible (or I don't know how). The reason that led me to put this here was because in my first approach without saving, an Exception would pop up due to conflicting ID references.

The existing [horse and raven drop groups id bug](https://github.com/MUnique/OpenMU/blob/0039043a331fc77319eb290282ecaf172519d531/src/Persistence/Initialization/VersionSeasonSix/Items/Pets.cs#L163) makes it that before saving here you will get an **equal ID** for the old horse drop group and the new raven drop group coexisting at the same time, thus triggering the exception.

Reply:
I'll think about this 😅 Probably we need another id schema for these groups